dash_bio/metadata.json

```diff
@@ -1763,6 +1763,135 @@
       }
     }
   },
+  "src/lib/components/Molecule2dViewer.react.js": {
+    "description": "The Molecule2dViewer component is used to render structural\nformulae of molecules.\nRead more about the component here:\nhttps://github.com/Autodesk/molecule-2d-for-react",
+    "displayName": "Molecule2dViewer",
+    "methods": [
+      {
+        "name": "onChangeSelection",
+        "docblock": null,
+        "modifiers": [],
+        "params": [
+          {
+            "name": "selectedAtomIds",
+            "type": null
+          }
+        ],
+        "returns": null
+      }
+    ],
+    "props": {
+      "id": {
+        "type": {
+          "name": "string"
+        },
+        "required": false,
+        "description": "The ID used to identify this component in callbacks"
```

Comment:
```suggestion
        "description": "The ID used to identify this component in callbacks."
```

Reply:
Oh, please transpose my comments to the source (`src/lib/components/Molecule2dViewer.react.js`).

Reply:
Fixed in 7bb2b00!